frontend/src/containers/githubOAuth/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './index.css';
+
+const SignInWithGithub = () => {
+  // const githubUrl = 'https://github.com/login/oauth/authorize?client_id=ed6fd12498c202cc79a0';
+  const handleButtonClick = () => window.open('http://localhost:3000/ghtoken', 'githubOuth');
```

Comment:
You can keep these URL's in a config file. Otherwise, you have to find all and replace all these.

Reply:
Agree on this. I will do it.